qutebrowser/config/configtypes.py

```diff
@@ -806,6 +818,10 @@ def validate(self, value):
                 raise configexc.ValidationError(value, "may not be empty!")
         value = os.path.expanduser(value)
         try:
+            if not os.path.isabs(value):
+                abspath = os.path.join(standarddir.config(), value)
+                if os.path.isfile(abspath):
+                    return
```

Comment:
Now that you mention it, the first check seems irrelevant. For the creation of ```abspath``` there is no other reason that coding style. I initially wanted to write it in one line like this:
```
if os.path.isfile(os.path.join(standarddir.config(), value)):
    return
```
But looking at the other code it seemed to me that a more verbose style was preferred (eg. two separate lines for expanduser and expandvars instead of ```os.path.expanduser(os.path.expandvars(...))```)